x/rollup/keeper/msg_server_test.go

```diff
@@ -128,16 +126,17 @@ func (s *KeeperTestSuite) TestApplyL1Txs() {
 
 				// Verify that the l1 block info and l1 block history are saved to the store
 				expectedBlockInfo := eth.BlockToInfo(testutils.GenerateL1Block())
+
 				l1BlockInfoBz := s.rollupStore.Get([]byte(types.KeyL1BlockInfo))
 				s.Require().NotNil(l1BlockInfoBz)
 
-				var l1BlockInfo *derive.L1BlockInfo
-				err = json.Unmarshal(l1BlockInfoBz, &l1BlockInfo)
+				l1BlockInfo := new(types.L1BlockInfo)
+				err = proto.Unmarshal(l1BlockInfoBz, l1BlockInfo)
```

Comment:
This causes a null pointer error, so i'll just use this
`l1BlockInfo := &types.L1BlockInfo{}`

Reply:
Oh yeah I think you could just use `var l1BlockInfo types.L1BlockInfo` instead of the pointer type and it would work as expected. I think using `l1BlockInfo := &types.L1BlockInfo{}` is good as well so no need for any more updates here